lib/tests/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ add_unit_test(CRITERION TARGET test_parse_number)
 add_unit_test(CRITERION TARGET test_reloc)
 add_unit_test(CRITERION TARGET test_hostname)
 add_unit_test(CRITERION LIBTEST TARGET test_rcptid)
-add_unit_test(CRITERION TARGET test_lexer)
+add_unit_test(LIBTEST TARGET test_lexer)
```

Comment:
You should not remove `CRITERION` from this test, as it is still criterion based.
The `CRITERION` and `LIBTEST` are kinda flags for `add_unit_test`, so you could apply both for the same test, and it means which should be linked/inlucded to the current test. In this case probably both of them should be applied.

Reply:
Done.